iris-mpc-cpu/src/hawkers/shared_irises.rs

```diff
@@ -132,13 +157,13 @@ impl<I: Clone> SharedIrises<I> {
     }
 
     pub fn last_vector_ids(&self, n: usize) -> Vec<VectorId> {
+        // Mihai: was this logic correct previously?
```

Comment:
It looks back at the last n insertions. There are odd cases where size<n, or where some vectors have been removed, then we return fewer than n items. Let me know if that’s wrong.

Reply:
It’s used by a feature called "LUC" if you’d like to poke around and double-check.

Reply:
FYI, Local Uniqueness Check, meaning time-local (last `n` here) and geography-local (provided by the upstream client as `luc_ids`).